feat: enhance layout and styling for better responsiveness

Ikou0504 opened the pull request:
- Update visionSignature style to display as block and adjust media query for smaller screens.
- Modify global paragraph styles to support pre-wrap and word break.
- Change main title to use a paragraph tag for improved formatting.
- Add word break to subtitle and link text for better readability.
- Update package dependencies to include style-to-object.

Commit:
feat: enhance layout and styling for better responsiveness

- Update visionSignature style to display as block and adjust media query for smaller screens.
- Modify global paragraph styles to support pre-wrap and word break.
- Change main title to use a paragraph tag for improved formatting.
- Add word break to subtitle and link text for better readability.
- Update package dependencies to include style-to-object.

okayama-daiki reviewed (changes requested):
お疲れ様です。レスポンシブデザインに関する修正をいただきありがとうございました。

何点か気になる点があったため、追加の修正をお願いします。

IidaKousei commented on app/page.css.ts:
```diff
@@ -1,6 +1,11 @@
 import { style, globalStyle } from "@vanilla-extract/css";
 import { vars } from "./theme.css";
 
+globalStyle('p', {
+  whiteSpace: 'pre-wrap',
+  wordBreak: 'keep-all',
+});
+
```
wordBreakの設定方法がいまいち分かりません(-_-;)

Commit:
feat: improve text formatting and layout in About page

- Added whiteSpace and wordBreak styles to visionTitle and introTitle for better text handling
- Updated visionTitle to include a word break element for improved readability
- Removed unnecessary global styles for paragraph text
- Cleaned up package.json by removing unused dependency

okayama-daiki reviewed (approved):
良さそうに見えます、ありがとうございました。
最後に変更があったファイルのフォーマットを忘れずにコミットしてください。

P.S. package-lock.json が変更されているので、元の状態に戻しておいてください。
`git checkout main -- package-lock.json`

IidaKousei commented:
ありがとう

Commit:
chore: update package-lock.json dependencies

- Removed unused dependencies: style-to-object and its nested inline-style-parser
- Updated inline-style-parser version to 0.1.1
- Ensured sharp dependency remains at version 0.33.5
